articles/quickstart/webapp/aspnet-core/v2/04-authorization.md

```diff
@@ -97,9 +107,9 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-At this point the you have integrated with the the [Role based authorization](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles) mechanism of ASP.NET Core, which means that your can ensure that a user belongs to a particular role by simply decorating your controller actions with the `[Authorize(Roles = ?)]` attribute.
+Because you have integrated your application with the [Role based authorization](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles) mechanism, you can check which role to assign to a user. Add the `[Authorize(Roles = ?)]` attribute to your controller action.
```

Comment:
Once again, you have completely changed the meaning of the original paragraph.

> you can check which role to assign to a user

is wrong! You are checking which roles are are assigned to a user, not which role to assign to them.

Change to something like

> You can use the [Role based authorization](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles) mechanism to ensure that only users who belong to a specific role can access certain actions. To do this, add the `[Authorize(Roles = ?)]` attribute to your controller action.